packages/botonic-plugin-flow-builder/src/handoff.ts

```diff
@@ -36,15 +42,19 @@ export async function doHandoff(
   // }
 
   await handOffBuilder.handOff()
+
+  if (flowBuilderPlugin.trackEvent) {
+    await flowBuilderPlugin.trackEvent(request, 'HANDOFF_SUCCESSFULL', {
```

Comment:
What about passing also the list of enums to track event?

Reply:
I think it is better to have default events that are always the same. Maybe we could pass a list optionally and if it is not passed we could use the default events.

Reply:
cool! 👌